package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "okx-api",
-  "version": "1.5.0",
+  "version": "1.5.1",
```

Comment:
Btw this should be a major release, given the breaking changes. Can you set the version to 2.0.0?